openpype/modules/deadline/plugins/publish/collect_default_deadline_server.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""Collect default Deadline server."""
+from openpype.modules import ModulesManager
```

Comment:
Maybe it would be better if we had global collector which would create `ModulesManager` in context instead of creating it in each plugin where is used.